pymc4/_model.py

```diff
@@ -32,7 +32,8 @@ def __init__(self, template, forward_context, template_args):
     def _evaluate(self):
         """Call the template function with the saved arguments."""
         args, kwargs = self._template_args
-        return self._template._func(*args, **kwargs)
+        self._template._func(*args, **kwargs)
+        return
```

Comment:
I think just get rid of the return, looks like a bug to me otherwise.

Reply:
Interesting, I usually prefer the explicit return, even if its none, because it helps me visually bookmark the end of functions.

If this is a strong preference I can do it in another PR

Reply:
Never thought of a blank "return" as being a "bookmarker", @canyon289. I learned something new today! That said, I think it isn't Pythonic convention to put a "blank return" the way you have. Is there something in the Zen of Python we can refer to?

Reply:
I don't think there's any strong reference I can point to than personal preference.

That being said my personal preference is not a great argument to keep it!